learn/tanssi/proxy-accounts.md

```diff
@@ -0,0 +1,202 @@
+
+Proxy accounts can also help you implement the principle of least privilege for access control. For example, if you have multiple team members, you can give them the minimum level of access required to carry out their duties via a specific type of proxy account. 
+
+This tutorial will walk you through configuring a proxy account on the Tanssi Dancebox TestNet specifically for balance transfers. Then, it will demonstrate performing a balance transfer using the newly created proxy.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To follow along with this tutorial, you will need to have:
+
+- [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/accounts){target=\_blank} open and connected to the Tanssi Dancebox TestNet
+- Create or have two accounts accessible in Polkadot.js Apps
+- At least one of the accounts will need to be funded with `DANCE` tokens
+
+If you need help importing your accounts into Polkadot.js Apps, please check out the [Connecting to Polkadot.js](/builders/interact/substrate-api/wallets/talisman/#connecting-to-polkadotjs){target=\_blank} guide.
+
+## General Definitions {: #general-definitions }
+
+When setting up a proxy account, a bond for the proxy is taken out of your free balance and moved to your reserved balance. The bond is required as adding a proxy requires on-chain storage space, and it is recalculated for each proxy you add or remove. The bond is returned to your free balance after all proxies are removed from your account.
+
+The deposit is calculated based on a deposit base and a deposit factor:
+
+- **Deposit base** - the amount to be reserved for an account to have a proxy list
+- **Deposit factor** - the additional amount to be reserved for every proxy the primary account has
+
+The equation for calculating the deposit is:
+
+```text
+deposit base + deposit factor * number of proxies
+```
+
+You can find each of the relevant variables for the Dancebox TestNet below. Note that these values are subject to change for Tanssi MainNet.
+
+|    Variable    |                       Value                        |
+|:--------------:|:--------------------------------------------------:|
+|  Deposit base  |  {{ networks.dancebox.proxy.deposit_base }} DANCE  |
+| Deposit factor | {{ networks.dancebox.proxy.deposit_factor }} DANCE |
+|  Max proxies   | {{ networks.dancebox.proxy.max_proxies }} proxies  |
+
+## Proxy Types {: #proxy-types }
+
+When creating a proxy account, you must choose a type of proxy that will define how the proxy can be used. The available options are:
+
+- **`Any`** - allows the proxy account to use any function supported by the proxy pallet. There is no filtering of calls.
```

Comment:
Although governance isn't a thing yet, there is still a governance proxy and it is shown on polkadot.js apps as an option. So we should at least add it here, but mention that it's disabled since there is no governance yet